apps/minaintyg/src/pages/Certificates/components/CertificateStatusBadge.tsx

```diff
@@ -0,0 +1,24 @@
+import { IDSBadge } from '@frontend/ids-react-ts'
+import { ComponentProps } from 'react'
+import { CertificateStatus } from '../../../schema/certificateList.schema'
+
+type BadgeTypes = ComponentProps<typeof IDSBadge>['type']
+
+const getBadgeType = (status: CertificateStatus): BadgeTypes => {
```

Comment:
Tests are missing

Reply:
Good call, I would rather test the rendered output from the component, but the `type` attributes from IDS is not correctly rendered in `jsdom`.  I exported the private `getBadgeType` function and tested that separate while treating the component as static with a snapshot test.